tsdb/chunkenc/bstream.go

```diff
@@ -257,3 +267,9 @@ func (b *bstreamReader) loadNextBuffer(nbits uint8) bool {
 
 	return true
 }
+
+// wrapper for the standard library's ReadUvarint to make it work
+// with our bstream.
+func (b *bstreamReader) readUvarint() (uint64, error) {
+	return binary.ReadUvarint(b)
```

Comment:
We're writing bits but read byte aligned, I think this should use readByte , [check the msb bit](https://protobuf.dev/programming-guides/encoding/#varints) and do another readByte if it's 1. The resulting 1 or 2 bytes need to go into a buffer and call binary.ReadUvarint() on it to mirror the write.

Reply:
Actually I was wrong, read through bstreamReader is bit aligned due to ReadUvaring using the b.ReadByte() interface.